common/scripts/setup_env.sh

```diff
@@ -65,30 +73,27 @@ fi
 
 # Build image to use
 if [[ "${IMAGE_VERSION:-}" == "" ]]; then
-  export IMAGE_VERSION=master-2022-03-17T19-05-13
+  IMAGE_VERSION=master-2022-03-17T19-05-13
```

Comment:
This change might require a tweak in the automation that occurs post a `tools` image build.

Reply:
Might actually be OK: https://github.com/istio/common-files/blob/master/bin/update-build-image.sh#L52